src/taco-kits/TacoKitMetadata.json

```diff
@@ -281,6 +279,90 @@
                     "version": "1.1.0"
                 }
             }
+        },
+        "tacokit@3.0.0": {
+            "cordova-cli": "5.4.1",
+            "releaseNotesUri": "http://go.microsoft.com/fwlink/?LinkId=722537",
+            "default": true,
+            "platforms": {
+                "android": {
+                    "version": "4.1.1"
+                },
+                "ios": {
+                    "version": "3.9.0"
```

Comment:
It's slightly weird that 2.0.0 has ios: 3.9.1 and here we go back to 3.9.0. Why are we using 3.9.0 here?